interface/stylesheet.dm

```diff
@@ -74,7 +74,7 @@ h1.alert, h2.alert		{color: #000000;}
 .announce				{color: #228b22;	font-weight: bold;}
 .boldannounce			{color: #ff0000;	font-weight: bold;}
 .minorannounce			{					font-weight: bold;  font-size: 3;}
-.greenannounce			{color: #00ff00;	font-weight: bold;}
+.greenannounce			{color: #008f00;	font-weight: bold;}
```

Comment:
Why this change? Did we use this for anything else?

Reply:
forgot to mention this sorry! this is used for awards (I think those are achievements?) but it is a very neon green at the moment which is unreadable!
![image](https://user-images.githubusercontent.com/84903631/123882422-126d7200-d947-11eb-9d93-54a89e104954.png)

Reply:
Fair enough--put it in the change log though. I don't really know what tag you'd use...but should be in there.

Reply:
make sure you check it in dark mode too